exercises/armstrong-numbers/.meta/solutions/armstrong_numbers.rb

```diff
@@ -0,0 +1,7 @@
+class ArmstrongNumbers
+  def self.include?(number)
+    exponent = number.digits.size
+
+    number.digits.sum { |digit| digit ** exponent } == number
```

Comment:
Don't use spaces around `**` as it emulates how we expect to see it (similar to x<sup>y</sup>, very close together), and makes it clear it isn't a typographical error due to style showing intent.

Reply:
I'll change it, but I strongly disagree.  I think operators should always have spaces in languages where it is possible, but I would not consider myself a real rubyist yet (, so I defer to your opinion).

Reply:
done

Reply:
In case it's a comfort: no spaces around exponents is recommended by one of the Ruby Styleguides. Not saying I like it, but I settle for consistency :-)
https://rubystyle.guide/#spaces-operators